src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;// Tan Matthew Simon Castaneda
+
+import duke.helper.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
```

Comment:
Great import statements. Clear to the reader on what Java packages you are using, helps improve the code quality 👍